gluten-core/src/main/resources/substrait/proto/substrait/algebra.proto

```diff
@@ -159,16 +159,18 @@ message ReadRel {
         DwrfReadOptions dwrf = 13;
         TextReadOptions text = 14;
         JsonReadOptions json = 15;
-     }
+      }
 
-     message partitionColumn {
+      message partitionColumn {
         string key = 1;
         string value = 2;
-     }
-     repeated partitionColumn partition_columns = 16;
+      }
+      repeated partitionColumn partition_columns = 16;
+
+      /// File schema
+      NamedStruct schema = 17;
 
-     /// File schema
-     NamedStruct schema = 17;
+      bool stream_input = 18;
```

Comment:
is it necessary ? we check if it is a iterator input by comparing the prefix string is equal to iterator

Reply:
it's not necessary, I add it before for debugging purposes, already removed.